internal/firefox.py

```diff
@@ -369,6 +369,13 @@ def collect_browser_metrics(self, task):
         page_data = self.run_js_file('page_data.js')
         if page_data is not None:
             task['page_data'].update(page_data)
+        if 'heroElementTimes' in self.job and self.job['heroElementTimes']:
+            logging.debug("Collecting hero element positions")
+            hero_elements = self.run_js_file('hero_elements.js')
```

Comment:
Doesn't it also need to run the selectors provided by the server as heroElements?  As best as I can tell it only ever collects the defaults specified by hero_elements.js.

Reply:
Thanks! Forgot that I was mulling over how to do this. Ended up injecting the custom hero element selectors into a global `window.__wptHeroElements` object -- is that ok with you?

Reply:
I can only verify in Firefox & Chrome, but I will change to that approach because I like it better :)